### PR TITLE
#262 Check if the user has HTTP blocked

### PR DESCRIPTION
**Pull request by joeywangzr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "chart.js": "^3.9.1",
     "class-variance-authority": "^0.4.0",
     "clsx": "^1.2.1",
+    "cors": "^2.8.5",
```
Comment (Ynng) [inline package.json:35]: I don't think we need cors in the dashboard repo.
Might be better to revert `package.json` and `package-lock.json` back to the original state

File: src/pages/login/CoreConnect.tsx [hunk 2]
```diff
@@ -65,8 +69,51 @@ const CoreConnect = () => {
       });
   };
 
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch('http://132.145.101.179:42069/');
```
Comment (Ynng) [inline src/pages/login/CoreConnect.tsx:75]: We want a more robust way of checking this.
Like, if my server goes down, people will get random popups

File: src/pages/login/CoreConnect.tsx [hunk 3]
```diff
@@ -65,8 +69,51 @@ const CoreConnect = () => {
...
+        if (!response.ok) {
+          setShowPopup(true);
+          console.log("fail");
```
Comment (Ynng) [inline src/pages/login/CoreConnect.tsx:78]: Remove debug prints

File: src/pages/login/CoreConnect.tsx [hunk 4]
```diff
@@ -65,8 +69,51 @@ const CoreConnect = () => {
...
+    };
+
+    fetchData();
+  }, []);
+
   return (
     <div className="flex w-[768px] max-w-full flex-col items-stretch justify-center gap-12 rounded-2xl px-12 py-14 @container">
+      <div>
+        {showPopup && (
+          <div>
```
Comment (Ynng) [inline src/pages/login/CoreConnect.tsx:97]: Remove unnecessary <div> wrapper

**Review by Ynng — CHANGES_REQUESTED** [review]

File: src/pages/login/CoreConnect.tsx (hunk 2 above)
Comment (joeywangzr) [reply src/pages/login/CoreConnect.tsx:75]: any suggestions

**Pull request closed without merging**